### ISSUE TITLE
Updated map with current (2017) municipality borders

### ISSUE THREAD

**Issue by TeemuKoivisto** [tomimick/mapcolorizer#2]
Hi,

as I was trying to make sense of this mess of creating the map of finnish municipalities I stumbled across your repository. Unfortunately I couldn't recreate the map the old way using the Latuviitta files as they didn't exist anymore. Also for some reason QGIS wouldn't even open your maps for lord knows why.

So I had to get creative and instead used another map with coastal borders http://datajournalismi.blogspot.com/2013/02/suomen-kuntarajat-2013-kml-formaatissa.html and then clipped the new municipality map with the old one removing excess area.

Long story short: I managed to do it and here it is: https://github.com/TeemuKoivisto/map-of-finland/blob/master/src/kuntarajat-2018.json

Also I've hosted an interactive version of it with React + react-simple-maps at the repo's Github pages: https://teemukoivisto.github.io/map-of-finland/

I'm not sure do you have time to update this or whatever you'll do but maybe someone might find this information useful one day.

Teemu
